File: Latihan3-py/Processor.py
class Processor:
    # Constructor
    def __init__(self):
        self.__name = ""
        self.__price = 0
    
    # Get-set Name
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name

    # Get-set Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

File: Latihan3-cpp/Processor.cpp
#include <string>

using namespace std;

class Processor{

    private:

    string name;
    int price;

    public:

    // Constructor
    Processor(){
    }

    Processor(string name, int price){
        this->name = name;
        this->price = price;
    }

    // Get-set Name
    void setName(string name){
        this->name = name;
    }

    string getName(){
        return this->name;
    }

    // Get-set Price
    void setPrice(int price){
        this->price = price;
    }

    int getPrice(){
        return this->price;
    }

    // Deconstructor
    ~Processor(){
    }

};